src/store.js

```diff
@@ -9,7 +9,6 @@ export const useAppStore = defineStore('app', {
       last_product_mode_used: 'barcode',
       last_currency_used: 'EUR',  // TODO: init with user locale ?
       recent_locations: [],
-      language: localStorage.getItem('user-locale') || import.meta.env.VITE_DEFAULT_LOCALE,  // 'en'
```

Comment:
we should keep the `user.language` key here + a default value (null or VITE_DEFAUL_LOCALE) + indicate it will be a code (ex : 'en')

Reply:
It should indeed be here. I messed up when merging the changes from master in 2nd commit and deleted the line.